src/FileDownloader.ts

```diff
@@ -58,20 +58,20 @@ export default class FileDownloader implements IFileDownloader {
         const timeoutInMs = settings?.timeoutInMs ?? DefaultTimeoutInMs;
         const retries = settings?.retries ?? DefaultRetries;
         const retryDelayInMs = settings?.retryDelayInMs ?? DefaultRetryDelayInMs;
-        try{
-            let progress = 0;
-            const progressTimerId = setInterval(() => {
-                if (progress <= 100) {
-                    // TODO: the whole timer should be under this if.
-                    if(onDownloadProgressChange != null){
-                        onDownloadProgressChange(progress++, 100);
-                    }
-                }
-                else{
-                    clearInterval(progressTimerId);
+        let progress = 0;
+        const progressTimerId = setInterval(() => {
+            if (progress <= 100) {
+                // TODO: the whole timer should be under this if.
+                if(onDownloadProgressChange != null){
+                    onDownloadProgressChange(progress++, 100);
                 }
-            }, 1500);
+            }
+            else{
```

Comment:
Nit: could you please add a space before the bracket? Same comment above/below for other occurrences. I don't believe that the linter is ran on this repo by default, so feel free to run it manually.

Reply:
Actually, I saw you're not the one who introduced it. Feel free to keep as is.